docs/BacwardCompatibility.md

```diff
@@ -0,0 +1,239 @@
+compatability is covered by the CRDs compatibility.
+- The `datacatalog.json`, `infraatributes.json` and `policymanager.json` files are generated form files in the 
+[pkg/model](https://github.com/fybrik/fybrik/tree/master/pkg/model), therefore any changes in the directory should be  
+validated for backward compatibility.
+- The basic [taxonomy file](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/taxonomy.json), is
+autogenerated from the both above sources. However, users are able to overwrite it, as describe by [Using a Custom 
+Taxonomy for Resource Validation](https://fybrik.io/v1.1/tasks/custom-taxonomy/). 
+
+Unfortunately all these files do not have version definition, and it can be complicated to separate files from different 
+releases. In order to be able to compare the files, I suggest to add an entry `version` and modify the 
+[json-schema-generator](https://github.com/fybrik/json-schema-generator) project to generate the entry. It's an optional 
+entry, so we don't have to change the go structures, but will be able to validate the content of the 
+`fybrik-taxonomy-config` configuration map. Due to possibility of user modifications of the basic taxonomy file we propose 
+in addition to the version, add the data of the file generation, and maybe its CRC. 
+
+*Note:* Should we create a taxonomy json file for the Credential Management connector?
+*Note:* Should we provide a [default metric taxonomy](https://fybrik.io/v1.1/tasks/infrastructure/#add-a-new-attribute-definition-to-the-taxonomy)
+
+## Default Policies
+Fybrik is installed with a set of different default policies. Most of the change in the policies do not prevent Fybrik 
+operation, but can change its default behaviour and therefore influence on its users.
+
+### Default Data Access Policies
+Fybrik can be deployed with different policy managers, for example it can be Open Policy Agent ([OPA](https://www.openpolicyagent.org/)), 
+or some other agent. However, usually any deployment contains some default setting, which can be all allowed, or all denied, 
+or other default policies settings. How it was set, depends on the used policy agent. When fybrik deployed with OPA, 
+the [default OPA policy rules](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/opa-server/policy-lib/default_policy.rego) 
+deployed as well and define Fybrik behavior if no other rules are not provided by users.
+
+### Default IT Config Policies 
+[IT config policies](https://fybrik.io/v1.1/concepts/config-policies/) are the mechanism via which the organization may 
+influence the construction of the data plane, taking into account infrastructure capabilities and costs.
+Out of the box policies come with the fybrik deployment. They define the deployment of basic capabilities, 
+such as read, write, copy and delete.
+The default IT policies and the default infrastructure definitions located at 
```

Comment:
are located

Reply:
The default IT policies and the default infrastructure definitions **exist** at